### PR TITLE
Porting old Umbraco API Controller [Relation Service] 

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description
This PR belongs to: https://github.com/umbraco/UmbracoDocs/issues/6466.

* [x] Change `UmbracoApiController` to `Controller`
* [x] Remove using statement `using Umbraco.Cms.Web.Common.Controllers;`
* [x] Add Attributes
* [x] Change `IComponent` to `NotificationHandler`

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version

## Product & version (if relevant)
Umbraco CMS v14 & 15

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks @erikjanwestendorp ! 💪 

I'll make sure we review this as soon as possible.

File: 15/umbraco-cms/reference/management/using-services/relationservice.md [hunk 1]
```diff
@@ -93,45 +71,45 @@ The example below uses UmbracoApiController which is obsolete in Umbraco 14 and
 {% endhint %}
```
Comment (sofietoft) [inline 15/umbraco-cms/reference/management/using-services/relationservice.md:71]: Should we remove this hint? 😄

**Review by sofietoft — COMMENTED** [review]
Thanks again @erikjanwestendorp !

I've made a single suggestion to remove the note before the Controller code snippet.

File: 15/umbraco-cms/reference/management/using-services/relationservice.md (hunk 1 above)
Comment (erikjanwestendorp) [reply 15/umbraco-cms/reference/management/using-services/relationservice.md]: @sofietoft oops.. 🙈

**Comment by erikjanwestendorp** [comment]
@sofietoft Just updated the PR with your suggestion 😄

**Pull request merged**